index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Tribute Page</title>
+</head>
+<body>
+    <main id="main" >
+        <h1 id="title">Dr. Norman Borlaug</h1>
+        <p id="title-description">The man who saved a billion lives</p>
```

Comment:
Please put the h1 and p tags inside a header for HTML semantic

Reply:
- [OPTIONAL] Nice one @Abdessittir , but try as much as you can to include `WHY` the student is asked to make use of a suggestion as this would make it clearer for the student to know why he or she is being asked to do a certain thing. And  if possible, you can also direct the student to a resource where more information could be got for  more clarification.

Reply:
Hi @Qoosim thanks for the feedback

Reply:
You're most welcome brother :heart: